CMakeLists.txt

```diff
@@ -81,12 +91,16 @@ target_include_directories(Vex PUBLIC
 )
 
 # DX12 project files
-set(VEX_DX12_SOURCES
-)
+if (WIN32 AND VEX_ENABLE_DX12)
```

Comment:
In this file the main changes are that I've made the VEX_DX12 and VEX_VULKAN macros always be defined, they will instead be set to 0 or 1 according to the user's environment.